### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Briancoughlin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,12 +3,17 @@
...
 | -- |
-| Boss Room: Small Scale Co-op Sample is built on top of the MLAPI package. The MLAPI package  is on the road to being a fully featured solution. We have solutions architects available on Discord and forums to help you work through issues you encounter. |
+| Boss Room: Small Scale Co-op Sample is built on top of the MLAPI package. 
 
 Boss Room is a fully functional co-op multiplayer RPG made with Unity MLAPI. It is built to serve as an educational sample that showcases certain typical gameplay patterns that are frequently featured in similar games.
 
 Our intention is that you can use everything in this project as a starting point or as bits and pieces in your own Unity games. The project is licensed under the Unity Companion License. See [LICENSE.md](LICENSE.md) for more legal information.
 
+> __IMPORTANT__:
+- Boss Room supports all major Unity platforms. To use the WebGL platform a custom WebGL transport based on web sockets is needed.
```
Comment (SamuelBellomo) [inline README.md:13]: we only support platforms supported by MLAPI
Windows and Mac

File: README.md [hunk 2]
```diff
@@ -3,12 +3,17 @@
 
 | 🛑  IMPORTANT - Early Access 🛑  | 
 | -- |
-| Boss Room: Small Scale Co-op Sample is built on top of the MLAPI package. The MLAPI package  is on the road to being a fully featured solution. We have solutions architects available on Discord and forums to help you work through issues you encounter. |
+| Boss Room: Small Scale Co-op Sample is built on top of the MLAPI package.
```
Comment (SamuelBellomo) [inline README.md:6]: why was this removed?

Comment (Briancoughlin) [reply README.md:6]: Because this is a readme,  i wasn't sure if that was relevant  information for the repo readme.

File: README.md (hunk 1 above)
Comment (Briancoughlin) [reply README.md:13]: MLAPI supports all major unity platforms, is Boss room different.

File: README.md (hunk 2 above)
Comment (Briancoughlin) [reply README.md:6]: you can put it back, im just not sure it belongs in this readme?

File: README.md (hunk 1 above)
Comment (SamuelBellomo) [reply README.md:13]: mmm last I heard from Jesse, we only support mac and windows. WebGL requires a TCP transport which we don't have and nothing was tested on mobile. Consoles are currently being discussed between Gio and the internal consoles team, but we can't say it's supported. Did you hear something different?

Comment (Briancoughlin) [reply README.md:13]: clarified in Slack I was mistaken

**Comment by Briancoughlin** [comment]
duplicated by Briancoughlin patch 2
